index.js

```diff
@@ -376,3 +387,7 @@ function requestPage(requestOptions) {
     })
   })
 }
+
+function isImage(url) {
+  return /\.(gif|png|svg|ico|mp3|jpg|jpeg)((\?|\&).+)?$/i.test(url)
```

Comment:
- [`mp3`](https://en.wikipedia.org/wiki/MP3)?
- This will fail if url has `#`. Example:   
    ```
    https://blogs.scientificamerican.com/blogs/cache/file/D37C6F64-E11B-413B-B4B9D36CF0D7C877.jpg?w=590&h=393#77AB39F8
    ```
- I'd use `URL` instead of regex.
Inspired from [`pathExtension`](https://developer.apple.com/reference/foundation/url/1780487-pathextension) and [`lastPathComponent`](https://developer.apple.com/reference/foundation/url/1780317-lastpathcomponent) in Swift [URL](https://developer.apple.com/reference/foundation/url), here is a ready to use implementation: 

    ```javascript

    URL.prototype.__defineGetter__('pathExtension', function() {
       // lastPathComponent could also be defined as a getter, but I will leave that up to you to decide.
       let lastPathComponent = this.pathname.split("/").pop()
       return lastPathComponent.includes(".") ? lastPathComponent.split(".").pop() : ""
    })
    
    new URL("https://domain.com/resource.jpg?q=search&id=100#abc.103").pathExtension
    // jpg

   // Usage in this function:
   let imageExtensions = ["gif", "png", "svg", "ico", "mp3", "jpg", "jpeg"] // Would define outside of the function.
   imageExtensions.includes(new URL(url).pathExtension)
    ```